### DISCUSSION TITLE
How to include both front and back of the same ID in one image

### DISCUSSION THREAD

**Discussion by shenjinwei** [community/community#51007] [GitHub Education]
My work ID has the school name on the front and my name and date on the back.

`Upload proof of your academic status` Only one image is supported

I tried 3 times, all rejected:
1, uploaded the front side of the ID card, the reason of rejection `Your document does not appear to include a date demonstrating current academic affiliation.`
2. Upload the back of the ID card,  the reason of rejection `The image you selected does not appear to contain your school name.`
3. Merge the front and back photos into one picture, the reason of rejection `The image you used appears to be edited. Manipulated images may not be used. 
`

I am confused，how can I include the front and back of the ID card in a picture that cannot be Manipulated.

**Comment by samus-aran** [comment]
Hi @shenjinwei 👋 

If possible we would recommend to take a photo of your academic ID alongside of another piece of evidence that includes a date demonstrating current academic affiliation. 

Let us know how you get on 👍
